roles/ceph-osd/tasks/bcache.yml

```diff
@@ -10,14 +10,14 @@
   shell: ceph osd crush add-bucket {{ hybrid_pool }} root
   register: result
   changed_when: result.stdout |search("added bucket")
-  run_once: true
+  when: inventory_hostname == (play_hosts | intersect(groups['ceph_osds_hybrid']))[0]
```

Comment:
Sorry, I am not sure I fully understand your mean.
I can't think of any difference between run_once and this solution, except that this solution resolve the issue I mentioned in the PR description.
Could you kindly show the example?